Common/src/main/java/com/blamejared/crafttweaker/impl/util/sequence/SequenceManager.java

```diff
@@ -0,0 +1,113 @@
+package com.blamejared.crafttweaker.impl.util.sequence;
+
+import com.blamejared.crafttweaker.api.util.sequence.Sequence;
+import com.blamejared.crafttweaker.api.util.sequence.SequenceType;
+
+import java.util.*;
+
+/**
+ * Manages {@link Sequence}s, handles adding, removing and ticking.
+ */
+public class SequenceManager {
```

Comment:
Is this class supposed to be API? Or is it something that only CT is ever meant to deal with?